pandas/tests/reshape/test_pivot_multilevel.py

```diff
@@ -250,3 +250,46 @@ def test_pivot_df_multiindex_index_none():
         columns=Index(["label1", "label2"], name="label"),
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    argnames=["index", "columns"],
+    argvalues=[("index", ["col", "value"]), (["index", "value"], "col")],
+    ids=["values-and-columns", "values-and-index"],
+)
+def test_pivot_table_multiindex_values_as_two_params(index, columns, request):
+    # GH#61292
+    data = [
+        ["A", 1, 50, -1],
+        ["B", 1, 100, -2],
+        ["A", 2, 100, -2],
+        ["B", 2, 200, -4],
+    ]
+    df = pd.DataFrame(data=data, columns=["index", "col", "value", "extra"])
+    result = df.pivot_table(values="value", index=index, columns=columns)
+    nan = np.nan
+    if request.node.callspec.id == "values-and-columns":
+        e_data = [
+            [50.0, nan, 100.0, nan],
+            [nan, 100.0, nan, 200.0],
+        ]
+        e_index = Index(data=["A", "B"], name="index")
+        e_cols = MultiIndex.from_arrays(
+            arrays=[[1, 1, 2, 2], [50, 100, 100, 200]], names=["col", "value"]
+        )
+        expected = pd.DataFrame(data=e_data, index=e_index, columns=e_cols)
```

Comment:
Can remove this line if you also dedent L293.

Reply:
Good catch 👍